components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpoint.java

```diff
@@ -189,4 +195,25 @@ private String generateKSNameFromDomainName(String tenantDomain) {
         String ksName = tenantDomain.trim().replace(".", "-");
         return (ksName + ".jks");
     }
+
+    /**
+     * This method generates the base64 encoded certificate list from a Certificate array
```

Comment:
`This method generates the base64 encoded certificate list from a Certificate array` -> This generates the base64 encoded certificate list from a Certificate array.`

Reply:
Comments should start with capital letter and ends with `.`

Reply:
The method comment was added maintaining the consistency of other method comments in the class.